header/actor-zeta/base/communication_module.hpp

```diff
@@ -90,6 +56,7 @@ namespace actor_zeta { namespace base {
         virtual void enqueue_base(message_ptr, executor::execution_device*) = 0;
 
         auto address_book(detail::string_view) -> address_t;
+        auto address_book() -> address_range_t;
```

Comment:
Do you really need it?

Reply:
address_range_t and this function is used to exclude sender from broadcastimg in case of resending messages between two interconnected supervisors, like in "spawn" example